pennylane/qnodes/base.py

```diff
@@ -158,11 +158,11 @@ class BaseQNode:
             and returning a tuple of measured :class:`~.operation.Observable` instances.
         device (~pennylane._device.Device): computational device to execute the function on
         mutable (bool): whether the circuit is mutable, see above
-        properties (dict[str, Any] or None): additional keyword properties for adjusting the QNode behavior
+        kwargs (dict[str, Any] or None): additional keyword arguments for adjusting the QNode behavior
```

Comment:
Typically, I don't think we include `kwargs` in the `Args:` list. Instead, we tend to do the following:

* If there are `kwargs` that can affect the properties of this QNode, we list them (alongside defaults) in a new `Keyword Args:` section below `Args:`.

* If there are no `kwargs` that can affect this QNode, we typically don't list it at all!

Reply:
In this case, there are the `vis_check` and the `par_check` keyword arguments, so we could have:
```python
Keyword Args:
    vis_check (float): ...
    par_check (float): ...
```
Note that with the keyword args section, we _do_ provide default values in the description, since sphinx will not be able to automatically parse the defaults as it does with Args:.

Reply:
Thanks so much! I've added these too (set the type to `bool` based on their usage)